package.json

```diff
@@ -36,6 +36,7 @@
     "react-native-sensitive-info": "6.0.0-alpha.9",
     "react-native-webview": "^11.3.1",
     "react-native-windows": "0.64.0",
```

Comment:
I _think_ we need 0.64.2 for some features, might want to set ^0.64.0 just in case